rasa/core/channels/console.py

```diff
@@ -142,6 +142,7 @@ def get_user_input(button_question: questionary.Question) -> Optional[Text]:
                 button_question = print_bot_output(response)
 
         num_messages += 1
+        await asyncio.sleep(0)  # Yield event loop for others coroutines
```

Comment:
Do you actually need this one? I tested it and it works also without?

Reply:
@wochinge Indeed, I tested and it worked with the last commits :thinking: . But without it, no other actions can be executed on the event loop until the end of the `while` which can lead to other problems in the future. 

Should I remove it?

Reply:
Let's keep it in. I think it makes sense to have it there.